### PR TITLE
Update SRG-OS-000073-GPOS-00041 for RHEL 9 STIG

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:
Update OCIL and add a fixtext

#### Rationale:
 RHEL 9 STIG

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_logindefs/rule.yml [hunk 1]
```diff
@@ -50,10 +50,22 @@ references:
     stigid@sle15: SLES-15-010260
     stigid@ubuntu2004: UBTU-20-010404
 
-ocil_clause: 'it does not'
+ocil_clause: '"ENCRYPT_METHOD" is not set to SHA512'
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_logindefs/rule.yml:53]:
```suggestion
ocil_clause: 'ENCRYPT_METHOD is not set to SHA512'
```
In `set_password_hashing_algorithm_libuserconf` the config key is not in quotes.

**Review by Mab879 — CHANGES_REQUESTED** [review]
One question about quoiting in the OCIL clause.

**Comment by jan-cerny** [comment]
I have removed quotes and separated the condition for password-auth config file and used XCCDF variable in fixtext.

**Review by Mab879 — APPROVED** [review]

**Comment by Mab879** [comment]
@jan-cerny It looks like this PR needs a rebase.

**Comment by jan-cerny** [comment]
rebased

**Pull request merged**